splinepy/bezier.py

```diff
@@ -165,6 +220,13 @@ def composition_derivative(self, inner, inner_derivative):
 
 
 class Bezier(BezierBase):
+    """
+    Bezier (Spline).
+
+    See :class:`.BezierBase` for more information on the theory of Bezier
+    splines.
+    """
```

Comment:
Please also move the init docstring here so the parameters are described in the documentation.

Reply:
sphinx default is not to document any dunder methods. I once added a workaround to document `__init__` and it seems that I removed it in 646741a. Would you rather just revive that or document all of them in class doctring?

Reply:
I don't think an explicit documentation of the `__init__` and other python specifics makes a lot of sense. I'd rather have the documentation within the class header.

Reply:
Okay, then I'll move the `__init__()` methods to the class doc-strings to resolve this.